src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -168,6 +170,46 @@ def _get_helm_version(ignore_errors):
         _handle_error(obsolete_ver_error, ignore_errors)
 
 
+def _get_notary_version(ignore_errors):
```

Comment:
_get_notary_version  and _get_helm_version have some similar code. Maybe you can write a more general one to get version and check version.